src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -65,7 +60,7 @@ namespace Microsoft.ML.Trainers
     /// <i>R</i> is approximated by the product of <i>P</i>'s transpose and <i>Q</i>. This trainer implements
     /// <a href='https://www.csie.ntu.edu.tw/~cjlin/papers/libmf/mf_adaptive_pakdd.pdf'>a stochastic gradient method</a> for finding <i>P</i>
     /// and <i>Q</i> via minimizing the distance between<i> R</i> and the product of <i>P</i>'s transpose and Q.</para>.
-    /// <para>For users interested in the mathematical details, please see the references below.</para>
+    /// <para>The underlying library used in ML.NET matrix factorization can be found on <a href='https://github.com/cjlin1/libmf'>a Github repository</a>. For users interested in the mathematical details, please see the references below.</para>
```

Comment:
>on <a href='https://github.com/cjlin1/libmf'>a Github repository</a> [](start = 86, length = 68)

on the <a href='[https://github.com/cjlin1/libmf](https://github.com/cjlin1/libmf)'>libmf Github repository</a>